nixos/lib/systemd-unit-options.nix

```diff
@@ -89,6 +89,11 @@ in rec {
         NixOS, setting this option instead causes the equivalent
         inverse `.wants` symlink to be present,
         establishing the same desired relationship in a stateless way.
+
+        Note that for user units (`systemd.user.<name>.*`)
+        the standard way to make a unit start by default
+        at login of the user `<name>`
+        is to set this option to `[ "default.target" ]`.
```

Comment:
I tried to rewrite this section a bit to move the more important bits a bit up: https://github.com/NixOS/nixpkgs/compare/master...Mic92:nixpkgs:doc/systemd-wantedBy?expand=1
Let me what you think.

Reply:
I will merge https://github.com/NixOS/nixpkgs/pull/199007 than once the CI finishes.